migrations/1676731511125_initialize.cjs

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable camelcase */
```

Comment:
This needs to be a CommonJS file even though our project is ESM, which is why it uses the `.cjs` extension